language/system-test/analyze.test.js

```diff
@@ -13,50 +13,103 @@
 
 'use strict';
 
-var analyzeExample = require('../../language/analyze');
+var uuid = require('node-uuid');
+var path = require('path');
+var storage = require('@google-cloud/storage')();
+var program = require('../analyze');
+
+var bucketName = 'nodejs-docs-samples-test-' + uuid.v4();
+var fileName = 'text.txt';
+var localFilePath = path.join(__dirname, '../resources/text.txt');
+var text = 'President Obama is speaking at the White House.';
+var options = {
+  type: 'text'
+};
 
 describe('language:analyze', function () {
-  it('should analyze sentiment in text', function (done) {
-    analyzeExample.main(
-      'sentiment',
-      'This gazinga pin is bad and it should feel bad',
-      function (err, result) {
-        assert.ifError(err);
-        assert(result);
-        assert(result.documentSentiment);
-        assert(result.documentSentiment.polarity < 0);
+  before(function (done) {
+    storage.createBucket(bucketName, function (err, bucket) {
+      assert.equal(err, null);
+      bucket.upload(localFilePath, done);
+    });
+  });
+
+  after(function (done) {
+    storage.bucket(bucketName).deleteFiles({ force: true }, function (err) {
+      assert.equal(err, null);
+      storage.bucket(bucketName).delete(done);
+    });
+  });
+
+  describe('analyzeSentiment', function () {
+    it('should analyze sentiment in text', function (done) {
+      program.analyzeSentiment(text, options, function (err, sentiment) {
+        assert.equal(null, err);
+        assert.equal(typeof sentiment, 'object');
+        assert.equal(typeof sentiment.polarity, 'number');
+        assert.equal(typeof sentiment.magnitude, 'number');
+        done();
+      });
+    });
+  });
+
+  describe('analyzeSentimentFromFile', function () {
+    it('should analyze sentiment in a file', function (done) {
+      program.analyzeSentimentFromFile(bucketName, fileName, options, function (err, sentiment) {
+        assert.equal(null, err);
+        assert.equal(typeof sentiment, 'object');
+        assert.equal(typeof sentiment.polarity, 'number');
+        assert.equal(typeof sentiment.magnitude, 'number');
+        done();
+      });
+    });
+  });
+
+  describe('analyzeEntities', function () {
+    it('should analyze entities in text', function (done) {
+      program.analyzeEntities(text, options, function (err, entities) {
+        assert.equal(null, err);
```

Comment:
Nit: I'm used to the constant being on the right for asserts, and that's the standard we seem to use elsewhere. Maybe a quick find-replace?